rego.go

```diff
@@ -80,7 +80,7 @@ func main() {
 				if imp == "C" {
 					continue
 				}
-				impPkg, err := build.Import(imp, "", 0)
+				impPkg, err := build.Import(imp, wd, 0)
```

Comment:
Hmm, perhaps it'd be better to use `pkg.Dir` here, that way `rego` can be run from another directory and it'll still work.

I think that'll be more correct.